src/components/application_manager/test/commands/mobile/set_audio_streaming_indicator_test.cc

```diff
@@ -0,0 +1,484 @@
+                const bool is_voice_communication,
+                const bool is_media) {
+    ON_CALL(*mock_app_, is_navi()).WillByDefault(Return(is_navi));
+    ON_CALL(*mock_app_, is_media_application()).WillByDefault(Return(is_media));
+    ON_CALL(*mock_app_, is_voice_communication_supported())
+        .WillByDefault(Return(is_voice_communication));
+  }
+
+  void MediaAppSetup() {
+    AppSetup(false, false, true);
+  }
+
+  void VoiceCommunicationAppSetup() {
+    AppSetup(false, true, false);
+  }
+
+  void NaviAppSetup() {
+    AppSetup(true, false, false);
+  }
+
+  void NonAppSetup() {
+    AppSetup(false, false, false);
+  }
+
+  void RunCommandExpectations(
+      MessageSharedPtr msg,
+      const char* info,
+      const mobile_apis::Result::eType mobile_code,
+      const bool success,
+      utils::SharedPtr<SetAudioStreamingIndicatorRequest> cmd) {
+    EXPECT_CALL(app_mngr_,
+                ManageMobileCommand(
+                    _, am::commands::Command::CommandOrigin::ORIGIN_SDL))
+        .WillOnce(DoAll(SaveArg<0>(&msg), Return(true)));
+
+    cmd->Run();
+
+    ResultCommandExpectations(msg, info, mobile_code, success);
+  }
+
+  void OnEventCommandExtectations(
+      MessageSharedPtr msg_mobile_response,
+      const char* mobile_info,
+      const mobile_apis::Result::eType mobile_code,
+      const bool success,
+      MessageSharedPtr msg_hmi_response,
+      utils::SharedPtr<SetAudioStreamingIndicatorRequest> cmd) {
+    const hmi_apis::Common_Result::eType hmi_code =
+        static_cast<hmi_apis::Common_Result::eType>(
+            (*msg_hmi_response)[am::strings::params][am::hmi_response::code]
+                .asInt());
+
+    Event event_ui(hmi_apis::FunctionID::UI_SetAudioStreamingIndicator);
+    event_ui.set_smart_object(*msg_hmi_response);
+
+    EXPECT_CALL(mock_message_helper_, HMIToMobileResult(hmi_code))
+        .WillOnce(Return(mobile_code));
+    EXPECT_CALL(app_mngr_,
+                ManageMobileCommand(
+                    _, am::commands::Command::CommandOrigin::ORIGIN_SDL))
+        .WillOnce(DoAll(SaveArg<0>(&msg_mobile_response), Return(true)));
+
+    cmd->on_event(event_ui);
+
+    ResultCommandExpectations(
+        msg_mobile_response, mobile_info, mobile_code, success);
+  }
+
+  sync_primitives::Lock lock_;
+  NiceMock<MockHmiInterfaces> hmi_interfaces_;
+  MockMessageHelper& mock_message_helper_;
+  MockAppPtr mock_app_;
+};
+
+TEST_F(SetAudioStreamingIndicatorRequestTest,
+       Run_NotRegisteredApplication_APPLICATION_NOT_REGISTERED) {
+  MessageSharedPtr msg_mobile = CreateFullParamsSO();
+  utils::SharedPtr<SetAudioStreamingIndicatorRequest> command =
+      CreateCommand<SetAudioStreamingIndicatorRequest>(msg_mobile);
+  MessageSharedPtr msg_mobile_response = CreateFullParamsSO();
+
+  const mobile_apis::Result::eType mobile_response =
+      mobile_apis::Result::APPLICATION_NOT_REGISTERED;
+  const bool success = false;
+  const char* mobile_info = "";
```

Comment:
@vlantonov 
removed mobile_info
Done in 1456832e723cd462c70ff1e55e4818cce6867ef7